configure.ac

```diff
@@ -21,7 +21,7 @@
 #
 
 AC_PREREQ([2.69])
-AC_INIT([cc-oci-runtime], [2.1.1])
+AC_INIT([cc-oci-runtime], [2.1.2])
```

Comment:
I'm wondering if the poc branch needs a rebase as this release is over a month old now?